eos/rare-b-decays/lambda-b-to-lambda1520-gamma-naive.cc

```diff
@@ -0,0 +1,81 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2022 Méril Reboud
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <eos/maths/power-of.hh>
+#include <eos/rare-b-decays/lambda-b-to-lambda1520-gamma-naive.hh>
+#include <eos/rare-b-decays/charm-loops.hh>
+#include <eos/utils/memoise.hh>
+
+#include <gsl/gsl_sf.h>
+
+namespace eos
+{
+    using namespace std::placeholders;
+
+    LambdaBToLambda1520GammaAmplitudes<tag::Naive>::LambdaBToLambda1520GammaAmplitudes(const Parameters & p, const Options & o) :
+        AmplitudeGenerator(p, o),
+        m_c(p["mass::c"], *this)
```

Comment:
Huh? Where is the charm mass usedin the ``naive factorisation`` approximation?

Reply:
It isn't :)